pgo/setup.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-img="quay.io/ianzhang366/postgre-ansible:latest"
+username="ianzhang366" # use your user name here
+img="quay.io/$username/postgre-ansible:latest"
```

Comment:
here `$username` is an external environment, should be in capital letters. It should also be documented somewhere - that the user must define `USERNAME`.

Reply:
oh, I see you define it in the script itself. Could you please change it to `USERNAME` and add instructions to define it before running `setup.sh`?

Reply:
done